MonoDevelop.MSBuild.Editor/MSBuildContentType.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
```

Comment:
Is this needed?

Reply:
Needed in `MSBuildCompletionCommitManager.cs` not in `MSBuildCompletionSource.cs`

Reply:
Can it be removed now?

---
In reply to: [311801939](https://github.com/mhutch/MonoDevelop.MSBuildEditor/pull/19#discussion_r311801939) [](ancestors = 311801939)